arch/arm/boot/dts/xilinx/ni-zynq.dtsi

```diff
@@ -235,38 +114,41 @@
 			#address-cells = <1>;
 			#size-cells = <0>;
 		};
+	};
 
-		spi0: spi@e0006000 {
-			compatible = "cdns,spi-r1p6";
-			status = "disabled";
-			reg = <0xe0006000 0x1000>;
-			interrupts = <0 26 4>;
-			clock-names = "ref_clk", "pclk";
-			clocks = <&clkc 25>, <&clkc 34>;
-			#address-cells = <1>;
-			#size-cells = <0>;
-		};
+};
 
-		spi1: spi@e0007000 {
-			compatible = "cdns,spi-r1p6";
-			status = "disabled";
-			reg = <0xe0007000 0x1000>;
-			interrupts = <0 49 4>;
-			clock-names = "ref_clk", "pclk";
-			clocks = <&clkc 26>, <&clkc 35>;
-			#address-cells = <1>;
-			#size-cells = <0>;
-		};
+&smcc {
+	status = "okay";
+};
 
-		usb0: usb@e0002000 {
-			usb-phy = <&usb_phy0>;
-		};
+&nfc0 {
+	status = "okay";
 
-		usb1: usb@e0003000 {
-			usb-phy = <&usb_phy1>;
-		};
+	nand@0 {
+		reg = <0>;
 	};
+};
+
+&i2c0 {
+	nicpld@40 {
+		compatible = "ni,cpld";
+		reg = <0x40>;
+	};
+
+	ds3231_rtc@68 {
+		compatible = "ds3232";
+		status = "disabled";
+		reg = <0x68>;
+	};
+};
+
+&usb0 {
+	usb-phy = <&usb_phy0>;
+};
 
+&usb1 {
+	usb-phy = <&usb_phy1>;
 };
```

Comment:
@gratian Regarding your comments on the usb-phy attributes, I modified them to be added to the usb nodes by referencing the usb phandles. Should we keep the original bindings?

Reply:
Split these (along with the same change for i2c0) into a separate commit.